packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/snapshot_control_pane.dart

```diff
@@ -27,29 +27,30 @@ class SnapshotControlPane extends StatelessWidget {
         return Row(
           mainAxisAlignment: MainAxisAlignment.spaceBetween,
           children: [
-            Row(
-              children: [
-                const SizedBox(width: defaultSpacing),
-                if (!isProcessing && current.heap != null) ...[
-                  _DiffDropdown(
-                    current: current,
-                    controller: controller,
-                  ),
-                  const SizedBox(width: defaultSpacing),
-                  SizedBox(
-                    width: _classFilterWidth,
-                    child: _ClassFilter(onChanged: controller.setClassFilter),
-                  ),
-                  const SizedBox(width: defaultSpacing),
-                  ToCsvButton(
-                    minScreenWidthForTextBeforeScaling:
-                        primaryControlsMinVerboseWidth,
-                    onPressed: current.downloadToCsv,
-                  ),
+            Padding(
```

Comment:
I think we want this padding to be on the outer row. Shouldn't we also have the same padding to the right? `EdgeInsets.symmetric(horizontal: defaultSpacing)`

Reply:
the cross at the right looks good. We do not want to mov eit.